conda-store-server/conda_store_server/app.py

```diff
@@ -364,6 +380,11 @@ def session_factory(self):
             url=self.database_url,
             poolclass=QueuePool,
         )
+
+        # Sets the default build_key_version value in the DB based on the config
+        import conda_store_server
+
+        conda_store_server._BUILD_KEY_VERSION = self.build_key_version
```

Comment:
This feels a bit hacky. Why do we need a global constant for this?

Reply:
There is a big comment about it in a different file. No other way to do it due to import cycles.

```
# Avoids a cyclic dependency between the orm module and the module defining
# CondaStore.build_key_version. Because the orm module is loaded early on
# startup, we want to delay initialization of the build_key_version field until
# it's been read from the config.
```